exercises/phone-number/src/test/java/PhoneNumberTest.java

```diff
@@ -39,25 +39,15 @@ public void validWhen11DigitsAndFirstIs1() {
     }
 
     @Ignore
-    @Test
+    @Test(expected = IllegalArgumentException.class)
     public void invalidWhenOnly11Digits() {
-        final String expectedNumber = "0000000000";
         final String actualNumber = new PhoneNumber("21234567890").getNumber();
```

Comment:
You don't need to make a variable here since you're not using it again. You could just do <code>new PhoneNumber("21234567890").getNumber()</code> :)